lib/customisation.nix

```diff
@@ -357,10 +357,12 @@ rec {
 
       Takes an attribute set `attrs` and returns a scope that extends the original scope.
 
-    - `overrideScope` (`(AttrSet -> AttrSet) -> scope`)
+    - `overrideScope` (`(scope -> scope -> AttrSet) -> scope`)
```

Comment:
I don't think it *only* accepts the overlay form though!

Reply:
It does. This is how `extends g f` works.